settings.gradle.kts

```diff
@@ -7,9 +7,10 @@ pluginManagement {
 }
 
 include(
-  "android-auto-app",
+  ":android-auto-app",
+  ":compose-app",
   ":app",
```

Comment:
Do you think it would be nice to name app modules similarly to extensions/plugins, e.g. make module type as a prefix instead of suffix? E.g. `app-compose` etc.